src/components/article-creator/ArticleCreator.tsx

```diff
@@ -281,11 +332,19 @@ function ArticleCreator({ className }: { className?: string }) {
           }
 
           if (block.type === "table") {
-            const updatedTable = await processTable(block.data);
+            const updatedTable = await processTable(block.data as TableData);
             block.data.content = updatedTable;
             return block;
           }
 
+          // Process images
+          if (block.type === "image" && block.data.url.startsWith("data:image/")) {
+            console.log("Blockdata", block.data);
+            const updatedImage = await processImage(block.data as ImageData);
```

Comment:
Is this a better solution than https://github.com/editor-js/image?

Reply:
that requires yarn and were running npm

Reply:
also it works well so like im fine w it